compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+''''exec "python3" "$0" "$@" #'''
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+
+
+def _is_image(version) : 
+    cmd = ['docker', 'images', f'onecc:{version}']
```

Comment:
Using `-q` option will do the trick:)

```bash
$ docker images -q onecc:1.21.0
d243dfsfeid # image id
$ docker images -q onecc:wrong_version
# return nothing
```
Or you can also use `docker inspect`.
```bash
$ docker inspect --type=image onecc:1.21.0
```

Reply:
> Are you saying that I should simply check the image in the way you mentioned?

Strictly speaking, no. You can use another way. I just recommend it for simple implementation. Even if you are considering the case various version comes in, the method `_image_exists` still seems to need a simple check. Otherwise, you should rename the method or make another method for SRP(Single Responsibility Principle).